### PR TITLE
Rollup

### PR DESCRIPTION
**Pull request by alexpricedev**

### CODE REVIEW

File: packages/open-ui/src/components/Navigation/Navigation.module.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import 'styles/index.scss';
+@import '../../styles/index.scss';
```
Comment (gurjit03) [inline packages/open-ui/src/components/Navigation/Navigation.module.scss:1]: @remotealex we are providing the PATH_SASS in the .env file, ryt ?

File: packages/open-ui/src/main.tsx [hunk 2]
```diff
@@ -1,3 +1,8 @@
-import { Button } from './components/Button';
-
-export { Button };
+export * from './components/AutoGrid';
```
Comment (gurjit03) [inline packages/open-ui/src/main.tsx:1]: should we use named exports here @remotealex for naming consistency as mentioned in our WLDM experience ?

**Review by gurjit03 — CHANGES_REQUESTED** [review]

File: packages/open-ui/src/components/Navigation/Navigation.module.scss (hunk 1 above)
Comment (alexpricedev) [reply packages/open-ui/src/components/Navigation/Navigation.module.scss:1]: Rollup doesnt recognise the `PATH_SASS` so i removed it for now!

File: packages/open-ui/src/main.tsx (hunk 2 above)
Comment (alexpricedev) [reply packages/open-ui/src/main.tsx:1]: These are named exported 👍

**Pull request merged**